yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Do we need to update this for a comment?

Reply:
I guess it is autogenerated, and not something we should edit. So yes? Unless there is some way of having it ignore comments?